### DISCUSSION TITLE
Need help with CRC32 not found for SNES Xenocrisis and MD Romeow and Julicat

### DISCUSSION THREAD

**Discussion by sprudel79** [sanni/cartreader#1015] [Need help]
Hi everyone,
I have a V5 reader and would like to dump the two mentioned games from the title.
Here are the log files:

```
OSCR HW5 V14.0

[+] Mega Drive/Genesis

[+] Game Cartridge


Cart Info
 
Name: ROMEOWJULICAT
bramCheck: 0000
Size: 8 MBit
ChkS: 0000
Sram: 2 KBit
 
Press Button...

[+] Read ROM


Saving to MD/ROM/ROMEOWJULICAT/37/...
[*******************]
Internal checksum...Error
0B0A

CRC32... 38BF19E6 -> Not found
```

and

```
OSCR HW5 V14.0

[+] Super Nintendo/SFC

[+] SNES/SFC cartridge


Searching database...
Checksum: 3AD1

Name: XENOCRISIS
Revision: 0
Type: LoROM FastROM
ICs: ROM 
ROM Size: 128 KB (4 banks)
Save Size: 0 KB
Checksum: 3AD1
Press Button...

[+] Read ROM


Saving to SNES/ROM/XENOCRISIS/36/...
[*******************]
Checksum... 3AD1 -> OK
CRC32... FE5B38F0 -> Not found
```

What am I supposed to do? Do I need to update e.g. the SNES.txt file manually?  
Any help is highly appreciated. Thank you!

**Comment by PsyK0p4T** [comment]
Hi.
Yes, both games are currently not into the txt files, so CRC32 check doesn't success.

About Xenocrisis, can you take a picture of the PCB ? I heard there is a special enhancement chip inside, so even if the rom dump is complete, you probably won't be able to run it elsewhere.

For Romeow and Julicat, can you force the rom size to 16 MBits and let us know the CRC32 ?

**Reply by sprudel79** [reply]
Thanks @PsyK0p4T, 
here's the picture of the PCB, I hope it is helpful:

![IMG_20240815_131416](https://github.com/user-attachments/assets/0d8262ce-dc4e-4c00-b3d9-850589f663e1)

Regarding Romeow and Julicat I will try to check this soon.

**Reply by sprudel79** [reply]
Ok, here we go:

```
OSCR HW5 V14.0

[+] Flashrom Programmer

[+] Self Test (3V)

[+] Mega Drive/Genesis

[+] Game Cartridge


Cart Info
 
Name: ROMEOWJULICAT
bramCheck: 0000
Size: 8 MBit
ChkS: 0000
Sram: 2 KBit
 
Press Button...

[+] Force ROM size


ROM Size: 1 Mbit

Rotate to Change
Press to Select

ROM Size: 2 Mbit

Rotate to Change
Press to Select

ROM Size: 4 Mbit

Rotate to Change
Press to Select

ROM Size: 8 Mbit

Rotate to Change
Press to Select

ROM Size: 12 Mbit

Rotate to Change
Press to Select

ROM Size: 16 Mbit

Rotate to Change
Press to Select
ROM Size: 16 Mbit
Press Button...

[+] Read ROM


Saving to MD/ROM/ROMEOWJULICAT/38/...
[*******************]
Internal checksum...Error
B094

CRC32... 46F895D5 -> Not found
```

Thanks!

**Reply by StanMOI** [reply]
looks like PCB using « Raspberry Pi SC0914 »

**Reply by PsychoFox11** [reply]
Yeah regarding XenoCrisis, the devs said:
"A framebuffer wouldn't be possible for fullscreen with multiple palettes on a SNES. We had to rework all the sprite/tile/map VRAM allocations, DMA etc for the game to fit within the SNES limits.
C compilers for SNES were also just too limited or flakey - if we had the time for a full assembly rewrite there is no reason why it wouldn't have been feasible.
So we leverage the co-processor to execute the core logic and push the data to fit within those limits. A LOT more time was spent trying to rework the game data to fit within the SNES's VRAM limits."

So in essence I think it is just pumping A/V data to the SNES and reading its controller inputs. I might be a little off, but I've always thought of this release as more of a parlor trick than 'Look what the SNES can do'. Not down on anyone having fun playing it, but it's more of a device than a SNES cart haha.
They also said anti-dumping was a major point of it, cause they didn't like how much the other versions were pirated. I think fans will support them, pirates wouldn't pay anyway, and they'd make more if they could do ROM sales for people that emulate, but what do I know? lol

**Reply by PsyK0p4T** [reply]
@sprudel79 are you on discord ? I would like to ask you for more pictures and infos about headers

**Reply by sprudel79** [reply]
> @sprudel79 are you on discord ? I would like to ask you for more pictures and infos about headers

Yes, I am...but I am currently on leave so I will text you again in about one week. Thanks

**Reply by sprudel79** [reply]
Hi @PsyK0p4T, I am back and have joined the community Discord server, let's continue checking things there, thank you.
